### PR TITLE
Merge Stage in Prod

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
     branches: [ giddh-2.0, beta-stage, beta-branch, production ]
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
-concurrency: prod-build
```
Comment (prateek-giddh) [inline .github/workflows/build.yml:11]: Ignore this file

File: .github/workflows/manual.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [macos-latest]
+        os: [windows-latest, macos-latest]
```
Comment (prateek-giddh) [inline .github/workflows/manual.yml:34]: Ignore this file

File: apps/web-giddh/src/app/daybook/daybook.component.html [hunk 3]
```diff
@@ -11,28 +11,26 @@
...
-                                *ngIf="currentCompanyBranches?.length > 2 && currentOrganizationType === 'COMPANY'">
+                            *ngIf="currentCompanyBranches?.length > 2 && currentOrganizationType === 'COMPANY'">
                                 <div class="form-group mb-0">
                                     <sh-select [options]="currentCompanyBranches" [(ngModel)]="currentBranch.uniqueName"
-                                        [showClear]="false" [ItemHeight]="33" width="100%"
-                                        (selected)="handleBranchChange($event)" [isFilterEnabled]="true"
-                                        [placeholder]="localeData?.select_entity" [fixedValue]="currentBranch?.alias">
+                                    [showClear]="false" [ItemHeight]="33" width="100%"
+                                    (selected)="handleBranchChange($event)" [isFilterEnabled]="true"
+                                    placeholder="localeData?.select_entity" [fixedValue]="currentBranch?.alias">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/daybook/daybook.component.html:19]: Revert this change

File: apps/web-giddh/src/app/contact/contact.component.html [hunk 4]
```diff
@@ -545,7 +537,7 @@ <h1 class="mr-b1">{{localeData?.type_message_body}}:</h1>
                                         <label></label>
                                         <input type="checkbox" (click)="selectAccount($event, account)"
                                             [(ngModel)]="account.isSelected"
-                                            [ngClass]="{'checkbox-visible': (selectedCheckedContacts.length || isMobileScreen), 'position-absolute': !selectedCheckedContacts.length}" />
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.html:548]: Class name getting replaced. Please check

File: apps/web-giddh/src/app/contact/contact.component.html [hunk 5]
```diff
@@ -783,7 +775,7 @@ <h1>{{commonLocaleData?.app_no_record_found}}</h1>
                                         <label></label>
                                         <input type="checkbox" (click)="selectAccount($event, account) "
                                             [(ngModel)]="account.isSelected "
-                                            [ngClass]="{ 'checkbox-visible': (selectedCheckedContacts.length || isMobileScreen), 'position-absolute': !selectedCheckedContacts.length} " />
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.html:786]: Class name getting replaced. Please check

File: apps/web-giddh/src/app/gst/filing/header/filing-header.component.html [hunk 6]
```diff
@@ -24,7 +24,7 @@
                         {{moment(currentPeriod.to, giddhDateFormat).format('DD MMM YYYY')}})
                     </ng-container>
                 </div>
-                <div class="font-xs text-muted text-middle font-normal">{{localeData?.outward_supplies_details}}</div>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/gst/filing/header/filing-header.component.html:27]: Class name getting replaced. Please check

File: apps/web-giddh/src/app/inventory/components/aside-pane/aside-pane.components.html [hunk 7]
```diff
@@ -8,16 +8,16 @@ <h3>Create</h3>
     <button id="back" (click)="backButtonPressed()" *ngIf="hideFirstScreen">Back</button>
 
     <div class="aside-body flexy-child vertical-align-middle aside-panel-ledger" *ngIf="!hideFirstScreen" vr-item>
-        <div class="pd-b3 text-center">
+        <div class="pb-3 text-center">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/inventory/components/aside-pane/aside-pane.components.html:11]: Class name getting replaced. Please check

File: apps/web-giddh/src/app/inventory/components/sidebar-components/stockList.component.ts [hunk 8]
```diff
@@ -59,7 +59,14 @@ export class StockListComponent implements OnInit, OnDestroy {
 
     public ngOnInit() {
         this.sub = this.route.params.pipe(takeUntil(this.destroyed$)).subscribe(params => {
-            this.groupUniqueName = params['groupUniqueName'];
+            if(params) {
+                this.groupUniqueName = params['groupUniqueName'];
+            }
+        });
+        this.breakPointObserver.observe([
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/inventory/components/sidebar-components/stockList.component.ts:66]: duplicate code. need to revert

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+                                <i class="icon-company mr-2"></i>
+                                <span class="company-name-right">{{ company.name }}</span>
+                                <span class="font-11 text-right branch-count pt-0 pb-0" *ngIf="company?.branchCount > 1">{{ company?.branchCount }}</span>
+                            </span>
+                        </li>
+                    </ul>
+                    <giddh-page-loader *ngIf="(isCompanyRefreshInProcess$ | async)"></giddh-page-loader>
+                </div>
+            </tab>
+            <tab [heading]="commonLocaleData?.app_branch" (selectTab)="tabChanged('branch')">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.html:52]: This file's code is not latest

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts [hunk 10]
```diff
@@ -0,0 +1,430 @@
...
+        }
+    }
+
+    /**
+     * This will get branches of company
+     *
+     * @param {*} company
+     * @memberof CompanyBranchComponent
+     */
+    public getCompanyBranches(design: number, company: any, reloadBranches?: boolean): void {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts:285]: This file's code is not latest

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 11]
```diff
@@ -8900,6 +8977,11 @@ body.update-scroll-hidden{
     }
 }
 @media (max-width: 767px) {
+    .gst-sidebar-open {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:8980]: this should be removed.

File: apps/web-giddh/webpack.electron.prod.partial.js [hunk 12]
```diff
@@ -9,7 +9,7 @@ module.exports = {
             'errlyticsNeeded': JSON.stringify(false),
             'errlyticsKey': JSON.stringify(''),
             'AppUrl': JSON.stringify('./'),
-            'ApiUrl': JSON.stringify('https://api.giddh.com/'),
+            'ApiUrl': JSON.stringify('https://apitest.giddh.com/'),
```
Comment (ravinderwalkover) [inline apps/web-giddh/webpack.electron.prod.partial.js:12]: prod url change should be reverted

File: apps/web-giddh/webpack.electron.prod.partial.js [hunk 13]
```diff
@@ -37,7 +37,7 @@ module.exports = {
             'process.env.errlyticsNeeded': JSON.stringify(false),
             'process.env.errlyticsKey': JSON.stringify(''),
             'process.env.AppUrl': JSON.stringify('./'),
-            'process.env.ApiUrl': JSON.stringify('https://api.giddh.com/'),
```
Comment (ravinderwalkover) [inline apps/web-giddh/webpack.electron.prod.partial.js:40]: prod url change should be reverted

File: apps/web-giddh/webpack.partial.js [hunk 14]
```diff
@@ -11,7 +11,7 @@ module.exports = {
             'errlyticsNeeded': JSON.stringify(false),
             'errlyticsKey': JSON.stringify(''),
             'AppUrl': JSON.stringify('http://localhost:3000/'),
-            'ApiUrl': JSON.stringify('https://api.giddh.com/'),
```
Comment (ravinderwalkover) [inline apps/web-giddh/webpack.partial.js:14]: prod url change should be reverted

File: apps/web-giddh/webpack.partial.js [hunk 15]
```diff
@@ -39,7 +39,7 @@ module.exports = {
             'process.env.errlyticsNeeded': JSON.stringify(false),
             'process.env.errlyticsKey': JSON.stringify(''),
             'process.env.AppUrl': JSON.stringify('http://localhost:3000/'),
-            'process.env.ApiUrl': JSON.stringify('https://api.giddh.com/'),
```
Comment (ravinderwalkover) [inline apps/web-giddh/webpack.partial.js:42]: prod url change should be reverted

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/contact/contact.component.html (hunk 4 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/contact/contact.component.html:545]: They do the same thing `position-absolute` is of Bootstrap and `pos-abs` is our custom. Will create a new PR that will replace all the class names with Bootstrap.

File: apps/web-giddh/src/app/contact/contact.component.html (hunk 5 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/contact/contact.component.html:779]: They do the same thing `position-absolute` is of Bootstrap and `pos-abs` is our custom. Will create a new PR that will replace all the class names with Bootstrap class in all the env.

File: apps/web-giddh/src/app/gst/filing/header/filing-header.component.html (hunk 6 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/gst/filing/header/filing-header.component.html:27]: Expected change confirmed with @shalinee-walkover

File: apps/web-giddh/src/app/inventory/components/aside-pane/aside-pane.components.html (hunk 7 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/inventory/components/aside-pane/aside-pane.components.html:11]: Confirmed from @aditi-tiwari17 it is required.

**Review by ravinderwalkover — APPROVED** [review]

**Review by prateek-giddh — COMMENTED** [review]
Done

**Review by shalinee-walkover — APPROVED** [review]

File: apps/web-giddh/webpack.cordova.local.partial.js [hunk 16]
```diff
@@ -10,7 +10,7 @@ module.exports = {
             'errlyticsNeeded': JSON.stringify(false),
             'errlyticsKey': JSON.stringify(''),
             'AppUrl': JSON.stringify('./'),
-            'ApiUrl': JSON.stringify('https://apidev.giddh.com/'),
+            'ApiUrl': JSON.stringify('https://apitest.giddh.com/'),
```
Comment (aditi-tiwari17) [inline apps/web-giddh/webpack.cordova.local.partial.js:13]: please check

File: apps/web-giddh/webpack.cordova.local.partial.js [hunk 17]
```diff
@@ -24,7 +24,7 @@ module.exports = {
             'process.env.errlyticsNeeded': JSON.stringify(false),
             'process.env.errlyticsKey': JSON.stringify(''),
             'process.env.AppUrl': JSON.stringify('./'),
-            'process.env.ApiUrl': JSON.stringify('https://apidev.giddh.com/'),
+            'process.env.ApiUrl': JSON.stringify('https://apitest.giddh.com/'),
```
Comment (aditi-tiwari17) [inline apps/web-giddh/webpack.cordova.local.partial.js:27]: please check

File: apps/web-giddh/webpack.cordova.stage.partial.js [hunk 18]
```diff
@@ -10,7 +10,7 @@ module.exports = {
             'errlyticsNeeded': JSON.stringify(false),
             'errlyticsKey': JSON.stringify(''),
             'AppUrl': JSON.stringify('./'),
-            'ApiUrl': JSON.stringify('https://apirelease.giddh.com/'),
+            'ApiUrl': JSON.stringify('https://apitest.giddh.com/'),
```
Comment (aditi-tiwari17) [inline apps/web-giddh/webpack.cordova.stage.partial.js:13]: please check

File: apps/web-giddh/webpack.cordova.stage.partial.js [hunk 19]
```diff
@@ -24,7 +24,7 @@ module.exports = {
             'process.env.errlyticsNeeded': JSON.stringify(false),
             'process.env.errlyticsKey': JSON.stringify(''),
             'process.env.AppUrl': JSON.stringify('./'),
-            'process.env.ApiUrl': JSON.stringify('https://apirelease.giddh.com/'),
+            'process.env.ApiUrl': JSON.stringify('https://apitest.giddh.com/'),
```
Comment (aditi-tiwari17) [inline apps/web-giddh/webpack.cordova.stage.partial.js:27]: please check

**Review by ravinderwalkover — APPROVED** [review]
